biscuit-auth/src/token/authorizer.rs

```diff
@@ -538,20 +543,25 @@ impl<'t> Authorizer<'t> {
     where
         error::Token: From<<R as TryInto<Rule>>::Error>,
     {
-        let rule = rule.try_into()?;
+        let rule = rule.try_into()?.convert(&mut self.symbols);
 
         self.world
             .run_with_limits(&self.symbols, limits.into())
             .map_err(error::Token::RunLimit)?;
-        let rule = rule.convert(&mut self.symbols);
-        let origin = if let Some(t) = self.token {
+        let all_origins = if let Some(t) = self.token {
             std::iter::repeat(())
```

Comment:
this should also contain the authorizer's scope no?

Reply:
It could be argued either way it think. Both are fine to me.

Reply:
If we see the query as a way to explore the final state of the world, compared to the authorizer whose role is to authorize or not the token, then not reusing the authorizer scope makes sense

Reply:
I think it should have access to the authorizer too. Qury should beable to explore the entire space, and the authorizer facts are part of it

Reply:
Ah i might have misunderstood, i was talking about using a default scope annotation